mesonbuild/modules/simd.py

```diff
@@ -13,76 +13,119 @@
 # limitations under the License.
 from __future__ import annotations
 
-from .. import mesonlib, compilers, mlog
+import typing as T
+from mesonbuild.interpreter.type_checking import STATIC_LIB_KWS, SOURCES_KW
+from mesonbuild.interpreterbase.decorators import KwargInfo, typed_pos_args, typed_kwargs
+
+from .. import mesonlib, mlog
 from .. import build
+from ..compilers import Compiler
```

Comment:
The order of the imports you're adding seems all over the place... and why are some of them `from mesonbuild.`

Reply:
In general, this commit seems to have leaned very heavily into avoiding cases where git can recognize previous content in the diff.